docs/overview.md

```diff
@@ -5,90 +5,77 @@ sidebar_label: Overview
+Replicated Volumes also are capable of enterprise storage features like snapshots, clone, volume expansion and so forth. Replicated Volumes are a preferred choice for Stateful workloads like Percona/MySQL, Jira, GitLab, etc. 
 
-- [Verify if OpenEBS is installed successfully](/docs/next/installation.html#verifying-openebs-installation)
-  and start provisioning OpenEBS volumes through Kubernetes PVC
-  interface by using `kubectl` command. 
+Depending on the type of storage attached to your Kubernetes worker nodes and application performance requirements, you can select from Jiva, cStor or Mayastor. 
+
+### Selecting the right storage engine
+
+See the following table for recommendation on which engine is right for your application depending on the application requirements and storage available on your Kubernetes nodes. 
+
+| Application requirements   | Storage Type | OpenEBS Volumes
+|--- |--- |--- 
+| Low Latency, High Availability, Synchronous replication, Snapshots, Clones, Thin provisioning | SSDs/Cloud Volumes | <a href="https://mayastor.gitbook.io/introduction/" target="_blank">OpenEBS Mayastor</a>
+| High Availability, Synchronous replication, Snapshots, Clones, Thin provisioning | Disks/SSDs/Cloud Volumes | <a href="https://github.com/openebs/cstor-operators" target="_blank">OpenEBS cStor</a>
+| High Availability, Synchronous replication, Thin provisioning | hostpath or external mounted storage | [OpenEBS Jiva](/docs/next/jivaguide.html)
+| Low latency, Local PV | hostpath or external mounted storage | [Dynamic Local PV - Hostpath](/docs/next/uglocalpv-hostpath.html)
+| Low latency, Local PV | Disks/SSDs/Cloud Volumes | [Dynamic Local PV - Device](/docs/next/uglocalpv-device.html)
+| Low latency, Local PV, Snapshots, Clones | Disks/SSDs/Cloud Volumes | <a href="https://github.com/openebs/zfs-localpv" target="_blank">OpenEBS Dynamic Local PV - ZFS </a>
+
+OpenEBS is also developing <a href="https://github.com/openebs/rawfile-localpv" target="_blank">Dynamic Local PV - Rawfile</a> storage engines available for alpha testing.
+
+OpenEBS has a vibrant community that can help you get started. If you have further question and want to learn more about OpenEBS, please join [OpenEBS community on Kubernetes Slack](https://kubernetes.slack.com). If you are already signed up, head to our discussions at [#openebs](https://kubernetes.slack.com/messages/openebs/) channel. 
 
 <br>
 
-## OpenEBS Storage Engines
+## Quickstart
 
-OpenEBS is a Kubernetes native hyperconverged storage solution. 
-OpenEBS consumes the storage (disks, SSDs, cloud volumes, etc) available 
-on the Kubernetes worker nodes to dynamically provision Kubernetes 
-Persistent Volumes. 
+Installing OpenEBS in your cluster is as simple as a few `kubectl` or `helm` commands depending on the storage engine. Here are the list of our Quickstart guides with detailed instructions for each storage engine.
```

Comment:
Installing OpenEBS in your cluster is as simple as a few `kubectl` or `helm` commands depending on the storage engine. -> Installing OpenEBS in your cluster is simple with the use of a few `kubectl` or `helm` commands depending on the storage engine that you are going to use.

Please ignore my comments if it is not simple.

Reply:
the current text reads better, conversational. thanks for reading through everything!